lib/api/src/grpc/proto/points.proto

```diff
@@ -603,6 +603,8 @@ message FacetCounts {
 message FacetValue {
     oneof variant {
         string string_value = 1; // String value from the facet
+        int64 integer_value = 2; // Integer value from the facet
+        string uuid_value = 3; // UUID value from the facet
```

Comment:
Why uuid is not just a `string_value`?

Reply:
Because this is the same structure we use for internal service, we need this to be able to deserialize it into uuid again. We could make a separate internal structure though

Reply:
Yes, let's make a separate structures for consistency with REST